The 5.36 bundle also disables the features "indirect", and
"multidimensional".

Indirect Object Syntax

Outside of the file handle case, use of this syntax is discouraged as it
can confuse the Perl interpreter. See below for more details.

Index: Coro/Socket.pm
--- Coro/Socket.pm.orig
+++ Coro/Socket.pm
@@ -62,6 +62,7 @@ For listening sockets, use C<AnyEvent::Socket::tcp_ser
 
 package Coro::Socket;
 
+use feature 'multidimensional';
 use common::sense;
 
 use Errno ();
@@ -194,7 +195,7 @@ sub configure {
          $! = 0;
 
          if ($self->connect ($_)) {
-            next unless writable $self;
+            next unless $self->writable();
             $! = unpack "i", $self->getsockopt (SOL_SOCKET, SO_ERROR);
          }
 
